src/openforms/js/compiled-lang/nl.json

```
@@ -5964,7 +5964,7 @@
   "ow5LZk": [
     {
       "type": 0,
-      "value": "Variables to include in the data to be sent."
+      "value": "Variabelen die worden toegevoegd aan de gegevens."
```

```suggestion
      "value": "Variabelen die aan de gegevens worden toegevoegd."
```
This is more personal preference I think.

---

Maybe, but it indeed sounds better 👍